test/unit/alignment/pairwise/alignment_result_debug_stream_test.cpp

```diff
@@ -36,44 +36,45 @@ TEST(alignment_result_test, debug_streamable)
     seqan3::debug_stream_type debug_stream{ostream};
 
     { // Print id and score
-        seqan3::detail::alignment_result_value_type result_value{id, score};
+        seqan3::detail::alignment_result_value_type result_value{id, id, score};
```

Comment:
I find it confusing that we have two times id here... Maybe explain it in a comment? Otherwise it looks like a mistake.

Reply:
Well, this is an intermediate fix for the tests. As it turned out we don't need the type deduction and also later we actually don't need the indirection of having a separate value type to set for the alignment_result. We changed that already for the search result. So at some point in the near future the whole test will be changed.